drivers/sensor/adxl345/Kconfig

```diff
@@ -3,10 +3,13 @@
 # Copyright (c) 2020 Antmicro <www.antmicro.com>
 # SPDX-License-Identifier: Apache-2.0
 
+DT_COMPAT_ADXL345 := adi,adxl345
+
 config ADXL345
 	bool "ADXL345 Three Axis accelerometer"
 	default y
 	depends on DT_HAS_ADI_ADXL345_ENABLED
 	depends on I2C || SPI
+	default $(dt_compat_enabled,$(DT_COMPAT_ADXL345))
```

Comment:
Not needed, this is happening with the `default y` and `depends on DT_HAS_ADI_ADXL345_ENABLED`.  Same comment for all the sensor Kconfig changes.

Reply:
Removed.